src/dataIntegration/transcriptomics/createTissueSpecificModel.m

```diff
@@ -1,701 +1,287 @@
-                Results.UnExpressed = [Results.UnExpressed;locus];
-
-            % Accounting for different probes and their binding positions
-            else
-                cntP = 0;
-                for j = 1:length(total)
-
-                    % Threshold once again, 75%
-                    if total(j) >= floor(0.75*size(ExpressionData.Data,2))*2;
-                        cntP = cntP+1;
-                    end
-                end
-
-                % If 50% or more of the probes have met the threshold,
-                % expressed
-                if cntP/length(total) >= 0.5
-                    Results.Expressed = [Results.Expressed;locus];
-                else
-                    Results.UnExpressed = [Results.UnExpressed;locus];
-                end
-            end
-        else
-
-            % If different RefSeq Accession codes, different transcripts,
-            % must be manually curated
-            Results.AltSplice = [Results.AltSplice;locus];
-        end
-        Results.Total = Results.Total+1;
-        clear total
-    end
-end
-
-% Setting up different transcripts for manual curation
-Transcripts.Locus = [];
-Transcripts.Data = {};
-for i = 1:length(Results.AltSplice)
-    num = find(Locus==Results.AltSplice(i));
-    transcripts = unique(ExpressionData.Transcript(num));
-    for j = 1:length(transcripts)
-        Transcripts.Locus = [Transcripts.Locus;Results.AltSplice(i)];
-    end
-    Transcripts.Data = [Transcripts.Data;transcripts];
-end
-
-% Determining each transcripts expression using a similar threshold as
-% before
-for i = 1:length(Transcripts.Data)
-    loc = strmatch(Transcripts.Data{i},ExpressionData.Transcript,'exact');
-    for j = 1:length(loc)
-        total(j) = sum(ExpressionData.Data(loc(j),:));
-        ExpressionData.Locus(loc(j)) = 0;
-    end
-    cap = length(loc)*11*2;
-    if sum(total)/cap >= ExpThreshold
-        Transcripts.Expression(i,1) = 1;
-    else
-        Transcripts.Expression(i,1) = 0;
-    end
-end
-
-% If expression of transcripts of one locus are the same, added as if no
-% alternative splicing occurs for simplicity
-Transcripts.Expressed = [];
-Transcripts.UnExpressed = [];
-mem_tran = Transcripts.Locus;
-for i = 1:length(Transcripts.Locus)
-    if Transcripts.Locus(i) > 0
-        locus = Transcripts.Locus(i);
-        loc = find(Transcripts.Locus==locus);
-        sumExp = 0;
-        for j = 1:length(loc)
-            sumExp = sumExp+Transcripts.Expression(loc(j));
-            Transcripts.Locus(loc(j)) = 0;
-        end
-        if sumExp == 0
-            Transcripts.UnExpressed = [Transcripts.UnExpressed;locus];
-        elseif sumExp == length(loc)
-            Transcripts.Expressed = [Transcripts.Expressed;locus];
-        end
-    end
-end
-Transcripts.Locus = mem_tran;
+function [tissueModel,Rxns] = createTissueSpecificModel(model, expressionData,exRxnRemove,solver,funcModel,options,param)
+                                                  
```

Comment:
If possible, please use an input parser for this method. 
e.g. I can imagine the most common call would be that someone wants to just set the solver and param, which now would lead to a call that sets two things just to their default. 
if you use an input-parser and parameter/value pairs, you can just set those parameters you want, and don't have to care about the rest.
maybe even merge the solver into the param input. (i.e. param.solver indicates which solver to use, and extract all useable fields afterwards).